packages/commerce-sdk-react/src/hooks/ShopperCustomers/queryKeyHelpers.ts

```diff
@@ -126,49 +127,49 @@ export type QueryKeys = {
 // This is defined here, rather than `types.ts`, because it relies on `Client` and `QueryKeys`,
 // and making those generic would add too much complexity.
 type QueryKeyHelper<T extends keyof QueryKeys> = {
-    /**
-     * Reduces the given parameters (which may have additional, unknown properties) to an object
-     * containing *only* the properties required for an endpoint.
-     */
-    parameters: (params: Params<T>) => Params<T>
+    // /**
+    //  * Reduces the given parameters (which may have additional, unknown properties) to an object
+    //  * containing *only* the properties required for an endpoint.
+    //  */
+    // parameters: (params: Params<T>) => Params<T>
     /** Generates the path component of the query key for an endpoint. */
     path: (params: Params<T>) => ExcludeTail<QueryKeys[T]>
     /** Generates the full query key for an endpoint. */
     queryKey: (params: Params<T>) => QueryKeys[T]
 }
 
-export const getExternalProfile: QueryKeyHelper<'getExternalProfile'> = {
-    parameters: (params) =>
-        pick(params, ['organizationId', 'externalId', 'authenticationProviderId', 'siteId']),
-    path: (params) => [
-        '/commerce-sdk-react',
-        '/organizations/',
-        params.organizationId,
-        '/customers/external-profile'
-    ],
-    queryKey: (params: Params<'getExternalProfile'>) => [
-        ...getExternalProfile.path(params),
-        getExternalProfile.parameters(params)
-    ]
-}
+// TODO: Re-implement (and update description from RAML spec) when the endpoint exits closed beta.
+// export const getExternalProfile: QueryKeyHelper<'getExternalProfile'> = {
+//     path: (params) => [
+//         '/commerce-sdk-react',
+//         '/organizations/',
+//         params.organizationId,
+//         '/customers/external-profile'
+//     ],
+//     queryKey: (params: Params<'getExternalProfile'>) => {
+//         const paramKeys = [
+//             ...paramKeysMap['getExternalProfile'],
+//             ...getCustomKeys(getCustomerBaskets)
+//         ]
+//         return [...getExternalProfile.path(params), pick(params, paramKeys)]
+//     }
+// }
```

Comment:
Whats going on here? Are we clawing back a change that was released, or is this something that hasn't been released yet. We want to keep an eye out for braking changes.